terraform/otel-ec2/main.tf

```diff
@@ -1,9 +1,18 @@
+locals {
+    // filter EC2 instances to launch
+    filtered_ec2_agents = length(var.ec2_filters) == 0 ? var.ec2_otels : {
+    for k, v in var.ec2_otels : k => v if anytrue([for f in var.ec2_filters :
+        strcontains(k, f)]) }
+
+    // Replace the TAG_OR_UNIQUE_NAME
+    assembled_ec2 = var.ec2_prefix == "" ? local.filtered_ec2_agents : { for k, v in local.filtered_ec2_agents : format("%s-%s", var.ec2_prefix, k) => v }
```

Comment:
Right now, we are using `%s:%s`  for prefix, aren't we?

Reply:
Fixed in https://github.com/newrelic-experimental/env-provisioner/pull/18/commits/104fe56e3c12f284e0a30049eea22e2cd7bc63ab